starter-code/css/main.css

```diff
@@ -0,0 +1,123 @@
+body {
+  font-family: "Lato", sans-serif;
+  margin: 30px 30px;
+  width: 600px;
+  height: auto;
+}
+
+header {
+  display: inline-block;
+  padding: 0;
+  line-height: 50%;
+}
+
+header nav a {
+  color: #eae2de;
+  text-align: left;
+  font-size: 20px;
+  margin: 10px 10px 0 0;
+  padding: 10px 0px 10px 0;
+  letter-spacing: 1px;
+  border-bottom: 1px solid #eae2de;
+}
+
+header nav a:hover {
+  font-size: 125%;
+  font-weight: 700;
+}
+
+h1, a {
+  color: #F8CE79;
+  text-decoration: none;
+  font-weight: 300;
+  margin-bottom: 0;
+}
+
+header h2 {
+  padding: 10px 0 0 0;
+}
+
+h2 {
+  font-weight: 300;
+  display: block;
+}
+
+h1 a { /* main heading/main title ONLY */
+  font-size: 48px;
+}
+
+h2 a { /* subheadings ONLY */
+  color: black;
+  font-size: 32px;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+article h3, h2 {
+  line-height: 50%;
+}
+
+article h3 {
+  color: #eae2de;
+  font-weight: 300;
+  font-size: 32px;
+}
+
+article p::first-letter {
+  float: left;
+  font-size: 5em;
+  margin: 0 .1em 0 0;
+  color: #eae2de;
+}
+
+article h4 a {
+  float: right;
+  display: inline;
+  font-weight: 700;
+  margin-bottom: 100px;
+}
+
+article {
+  padding: 20px 0 40px 0;
+  border-bottom: 1px solid #eae2de;
+}
+
+nav a {
+  display: inline-block;
+  text-decoration: underline;
+  font-size: 16px;
+  color: #F8CE79;
+}
+
+footer {
+  padding: 50px 0 0 0;
+  color: #eae2de;
+}
+
+footer nav {
+  color: #F8CE79;
+}
+
+footer, nav a {
+  font-size: 16px;
+  text-decoration: none;
```

Comment:
You've started to repeat this property throughout the file. Consider creating an anchor `a {  }` selector at the beginning of the file that clears this text-decoration to help keep your code DRY.